src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Date.txt

```diff
@@ -225,8 +249,35 @@ Date(2001,2,16)
 >> Date(2022, 1, 15) - Date(2022, 1, 30)
 -15
 
->> Date(2022, 1/0, 15) - Date(2022, 1, 30)
-#Error(Kind=Div0)
+>> Date(2022, 10, 1) + Time(12, 34, 56)
```

Comment:
>Time(12, 34, 56) [](http://example.com/codeflow?start=23&length=16)

Try with Time(30, 0, 0) as well

Reply:
Great idea - it found a bug in our Time implementation. Excel wraps it to be the same as Time(6,0,0), we weren't doing it. Fixed it.